packages/terra-application-links/src/tabs/_CollapsedTab.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import { Route, matchPath } from 'react-router-dom';
+import TabUtils from './_TabUtils';
+import styles from './ApplicationTabs.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Indicates if the tab is be to desplayed in the tab menu.
+   */
+  isCollapsed: PropTypes.bool,
+  /**
+   * The path to push to the route.
+   */
+  path: PropTypes.string.isRequired,
+  /**
+   * The display text for the tab.
+   */
+  text: PropTypes.string.isRequired,
+  /**
+   * The click callback of the tab..
+   */
+  onTabClick: PropTypes.func,
+};
+
+const CollapsedTab = ({
+  onTabClick,
+  path,
+  text,
+  ...customProps
+}) => (
+  <Route
+    render={({ location, history }) => {
+      const isActive = !!matchPath(location.pathname, { path });
+      const tabClassNames = cx([
+        'collapsed-tab',
+        { 'is-disabled': isActive },
```

Comment:
how does isActive map to is-disabled styling?

Reply:
I might be able to remove this, I'll look into it.